src/libraries/Common/src/System/IO/Compression/ZLibNative.cs

```diff
@@ -98,7 +98,11 @@ public enum CompressionLevel : int
         /// </summary>
         public enum CompressionStrategy : int
```

Comment:
Instead of using new `ZLibCompressionStrategy` had to keep internal enum and native API shape as is because it is referenced from `System.Net.WebSockets` project